src/views/LeadAdd/ProcessingLeads/ProcessingLeadsModal/index.js

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback, useContext, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { _cs } from '@togglecorp/fujs';
+
+import DangerButton from '#rsca/Button/DangerButton';
+import DangerConfirmButton from '#rsca/ConfirmButton/DangerConfirmButton';
+import PrimaryButton from '#rsca/Button/PrimaryButton';
+import Button from '#rsca/Button';
+import ListView from '#rscv/List/ListView';
+import Modal from '#rscv/Modal';
+import ModalHeader from '#rscv/Modal/Header';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+
+import { LeadProcessorContext } from '../../LeadProcessor';
+import {
+    leadKeySelector,
+    leadFaramValuesSelector,
+} from '../../utils';
+import LeadListItem from '../../LeadListItem';
+
+import styles from './styles.scss';
+
+const propTypes = {
+    className: PropTypes.string,
+    closeModal: PropTypes.func,
+    onLeadsAdd: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+    className: undefined,
+    closeModal: undefined,
+};
+
+function ProcessingLeadsModal(props) {
+    const {
+        className,
+        onLeadsAdd,
+        closeModal,
+    } = props;
+
+    const {
+        clearProcessingLeads,
+        processingLeads,
+        fileUploadStatuses,
+        driveUploadStatuses,
+        dropboxUploadStatuses,
+        removeProcessingLead,
+    } = useContext(LeadProcessorContext);
+
+    const processingLeadsRendererParams = useCallback((key, data) => {
+        const handleLeadRemove = () => removeProcessingLead(key);
+
+        const actionButtons = (
+            <Button
+                className={styles.button}
+                onClick={handleLeadRemove}
+                iconName="delete"
+            />
+        );
+
+        return ({
+            title: leadFaramValuesSelector(data)?.title,
+            lead: data,
+            progress: fileUploadStatuses?.[key] ? fileUploadStatuses[key].progress : undefined,
```

Comment:
```suggestion
            progress: fileUploadStatuses?.[key].progress,
```
Why not just read the value directly?